etcd/etcd-cluster-runtime-reconfiguration-on-coreos.md

```diff
@@ -0,0 +1,187 @@
+# etcd Cluster Runtime Reconfiguration on CoreOS
+
```

Comment:
ok, we just need to put whatever describes it.

"Use this document to reconfigure or recover an etcd cluster running on CoreOS, using a combination of systemd features and etcdctl commands."